### PR TITLE
Fix windows paths

### PR DESCRIPTION
**Pull request by robinryf**

### CODE REVIEW

File: AssetStoreBatchMode/AssetStoreBatchMode.cs [hunk 1]
```diff
@@ -325,7 +328,10 @@ static string GetDraftAssetsPath(string localRootPath)
     }
     static void Export(Package package, string localRootPath, string toPath)
     {
-        File.Delete(toPath);
+        if (File.Exists(toPath))
```
Comment (lmontoute) [inline AssetStoreBatchMode/AssetStoreBatchMode.cs]: Not sure this check is actually needed - from File.Delete docs

> If the file to be deleted does not exist, no exception is thrown.

Were there problems here for you?

File: AssetStoreBatchMode/AssetStoreBatchMode.cs [hunk 2]
```diff
@@ -203,6 +203,9 @@ public static void UploadAssetStorePackage(string username, string password, str
         // Validate root project folder
         var projectFolder = Path.Combine(Application.dataPath, s_RootPath ?? string.Empty);
 
+        // Convert to unix path style
+        projectFolder = projectFolder.Replace("\\", "/");
```
Comment (lmontoute) [inline AssetStoreBatchMode/AssetStoreBatchMode.cs]: Good catch, thanks!

File: AssetStoreBatchMode/AssetStoreBatchMode.cs (hunk 1 above)
Comment (robinryf) [reply AssetStoreBatchMode/AssetStoreBatchMode.cs]: I thought I had a problem with that piece of code. I just tried to reproduce it without my change but I couldn't. Maybe I had some file handle open on it. You are right, it does not throw an exception, my bad. I removed my change.
By the way, thanks for this repo! ;) Really helped me to automate the asset store package upload.

**Comment by lmontoute** [comment]
👍

**Pull request merged**